test/OpenApiClientTests/OpenApiClientTests.csproj

```
@@ -28,56 +29,56 @@
       <ClassName>OpenApiClient</ClassName>
       <OutputPath>OpenApiClient.cs</OutputPath>
       <CodeGenerator>NSwagCSharp</CodeGenerator>
-      <Options>/GenerateClientInterfaces:true /ClientClassAccessModifier:internal /GenerateExceptionClasses:false /AdditionalNamespaceUsages:JsonApiDotNetCore.OpenApi.Client.Exceptions</Options>
+      <Options>/GenerateClientInterfaces:true /ClientClassAccessModifier:internal /GenerateExceptionClasses:false /GenerateOptionalParameters:true /AdditionalNamespaceUsages:JsonApiDotNetCore.OpenApi.Client.Exceptions</Options>
```

I'd rather not have this switch added in test projects. While it leads to shorter code, it also hides potential errors when a new parameter gets added that wasn't intended.